homeassistant/components/proxmoxve/binary_sensor.py

```diff
@@ -66,7 +69,7 @@ def create_binary_sensor(coordinator, host_name, node_name, vm_id, name):
     return ProxmoxBinarySensor(
         coordinator=coordinator,
         unique_id=f"proxmox_{node_name}_{vm_id}_running",
-        name=f"{node_name}_{name}_running",
+        name=f"{node_name}_{name}",
```

Comment:
Just a quick follow up to make sure I'm not causing some headaches for people. Would changing the name value break anything in the UI? I know unique_id might. Figured I would ask. @epenet

Reply:
As long as a `unique_id` is defined, a name change is not considered a breaking change.
However, if a `unique_id` isn't defined on an entity, then it becomes a breaking change.

Reply:
Thank you for the clarification